cmd/timestamp-server/app/root.go

```diff
@@ -70,7 +70,8 @@ func init() {
 	rootCmd.PersistentFlags().String("file-signer-key-path", "", "Path to file containing PEM-encoded private key. Supported formats include PKCS#1, PKCS#8, and RFC5915 for EC")
 	rootCmd.PersistentFlags().String("file-signer-passwd", "", "Password to decrypt private key")
 	// NTP time introspection
-	rootCmd.PersistentFlags().String("ntp-monitoring", "", "Path to a file configuring ntp monitoring")
+	rootCmd.PersistentFlags().String("ntp-monitoring", "ntpsync.yaml", "Path to a file configuring ntp monitoring. Uses default ntpsync.yaml configuration if none is provided")
```

Comment:
I would probably change this to default to an empty string, and document if no string is provided, then we use the embedded configuration.